contracts/governance/README.adoc

```diff
@@ -64,6 +65,8 @@ Operations status can be queried using the functions:
 [[timelock-roles]]
 ==== Roles
 
+Proposer and Executor roles can be granted to the zero address to open them up and allow anyone to act as such a role.
```

Comment:
Its only true of executor. We removed that feature from Proposer for security reasons (proposers are also cancelors)